packages/modules/manage-versions/src/components/ManageVersions/VersionsTab/VersionsTab.tsx

```diff
@@ -143,8 +170,19 @@ const VersionsTab = (props: VersionsTabProps) => {
     [renderDateColumn]
   );
 
+  const toggleVersionState = useCallback(
+    (version: NamedVersion) => {
+      collapseAllRows();
+      handleHideVersion(version);
+    },
+    [handleHideVersion, collapseAllRows]
+  );
+
   const getToolbarActions = useCallback(
     (props: CellProps<VersionTableData>) => {
+      const version = props.row.original.version;
```

Comment:
you added here variable, but not reusing in all other place, which still use `props.row.original.version`.
Need to be consistent in a scope of project, not just a new code addition..